src/versions.rs

```diff
@@ -56,18 +56,17 @@ pub async fn get_version_mapping_with_fallback(
     }
 }
 
-fn version_to_url(
-    base_url: &str,
-    version: &str,
-    source: &str,
-) -> String {
+fn version_to_url(base_url: &str, version: &str, source: &str) -> String {
     let version = if version.starts_with("stable") {
         version.into()
     } else {
         format!("release-crates-io-v{}", version)
     };
 
-    format!("{}/paritytech/polkadot-sdk/{}/{}", base_url, version, source)
+    format!(
+        "{}/paritytech/polkadot-sdk/{}/{}",
```

Comment:
Revert please to keep the diff small.

Reply:
Do I still show the logs for which dependencies need a version upgrade or do I just exit saying there are dependencies that need update?

Reply:
If the tool already prints that for free then yea it can be helpful. But adding it additionally for now is not needed i think.

Reply:
I reverted back and made minimal changes @ggwpez